12/umbraco-cms/fundamentals/setup/install/umbraco-on-linux-macos.md

```diff
@@ -21,10 +21,10 @@ With the templates installed on your system, it is now possible to create Umbrac
 To create a project, there are two options:
 
 - Continue creating projects using the .NET CLI.
-- Create new projects using Visual Studio.
+- Create new projects using Visual Studio. (only macOS)
 
 To create new projects using Visual Studio, you can use the [Install using Visual Studio](visual-studio.md) guide.
 
 Once you create a new project it will use SQLite by default on Mac.
```

Comment:
Do you happen to know if SQLite is the default for Linux as well? 😅 
Otherwise, I'll run it by our developers. Let em know 😁